Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- TOC entry 209 (class 1259 OID 17752)
+-- Name: MDVFragments_from_excel; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MDVFragments_from_excel" (
+    met_id text NOT NULL,
+    formula text NOT NULL,
+    elements text[] NOT NULL,
+    positions integer[] NOT NULL,
+    id integer NOT NULL
+);
+
+
+ALTER TABLE public."MDVFragments_from_excel" OWNER TO postgres;
+
+--
+-- TOC entry 208 (class 1259 OID 17750)
+-- Name: MDVFragments_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."MDVFragments_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."MDVFragments_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3135 (class 0 OID 0)
+-- Dependencies: 208
+-- Name: MDVFragments_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."MDVFragments_id_seq" OWNED BY public."MDVFragments_from_excel".id;
+
+
+--
+-- TOC entry 207 (class 1259 OID 17743)
+-- Name: MFA model metabolites ; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MFA model metabolites " (
+    id integer NOT NULL,
+    mapping_id text NOT NULL,
+    met_id text NOT NULL,
+    met_elements text[],
+    met_atompositions integer[],
+    met_symmetry_elements text[],
+    met_symmetry_atompositions integer[],
+    used boolean,
+    comment text,
+    met_mapping json,
+    base_met_ids text[],
+    base_met_elements json,
+    base_met_atompositions json,
+    base_met_symmetry_elements json,
+    base_met_symmetry_atompositions json,
+    base_met_indices integer[]
+);
+
+
+ALTER TABLE public."MFA model metabolites " OWNER TO postgres;
+
+--
+-- TOC entry 206 (class 1259 OID 17741)
+-- Name: MFA model metabolites _id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."MFA model metabolites _id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."MFA model metabolites _id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3136 (class 0 OID 0)
+-- Dependencies: 206
+-- Name: MFA model metabolites _id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."MFA model metabolites _id_seq" OWNED BY public."MFA model metabolites ".id;
+
+
+--
+-- TOC entry 204 (class 1259 OID 17729)
+-- Name: MFA model reactions table; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MFA model reactions table" (
```

Comment:
Let's remove the white spaces and also the `table`

Reply:
done